egs2/swbd_sentiment/asr1/conf/tuning/train_asr_conformer_wav2vec2.yaml

```diff
@@ -0,0 +1,89 @@
+# network architecture
```

Comment:
The wav2vec2 and hubert confs are almost the same, can you combine them by adding comments instead of a new one? As for initial PR, we would recommend to simplify the config.